lib/shared/drive-constraints.ts

```diff
@@ -75,16 +86,16 @@ export function isSourceDrive(
  */
 export function isDriveLargeEnough(
 	drive: DrivelistDrive | undefined,
-	image: Image,
+	image: SourceMetadata,
 ): boolean {
-	const driveSize = _.get(drive, 'size') || UNKNOWN_SIZE;
+	const driveSize = drive?.size || UNKNOWN_SIZE;
 
-	if (_.get(image, ['isSizeEstimated'])) {
+	if (image?.isSizeEstimated) {
```

Comment:
No need for `?.` as image can't be undefined.

Reply:
It is undefined when you have none selected (default app state at startup)

Reply:
I made it so that the image is an empty object instead & changed the other checks accordingly, so now it follows the same behaviour as the drives (which is an empty list when not selected)